helm/kubernetes-coredns-chart/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: 1.1.1
 description: A Helm chart for CoreDNS
 name: kubernetes-coredns-chart
-version: 0.2.2-[[ .SHA ]]
+version: 0.2.3-[[ .SHA ]]
```

Comment:
There is no need to bump the patch version here.

Reply:
Correct, but also doesn't hurt :+1: